cmd/cloudshell_open/main.go

```diff
@@ -311,15 +316,13 @@ func checkCloudShellTrusted() (bool, error) {
 	return false, fmt.Errorf("error determining if cloud shell is trusted: %+v. output=\n%s", err, string(b))
 }
 
-func waitForBilling(projectID string, prompt func(string)) error {
-	for {
```

Comment:
looks like you've removed the for-loop here. why?

Reply:
I thought this was unintentional, but is this intentional for waiting until billing will be enabled and is infinite loop OK if billing will not be enabled?

Reply:
If this loop is intentional, I will revert loop removal and return result of prompt only when error happened, of course!

Reply:
Yeah this is intentional. It would be good to preserve the logic.

Reply:
Thank you! I'll fix!